chapters/offenses.adoc

```diff
@@ -105,7 +123,9 @@ NOTE: This rule does not use the standard sanctions defined for <<Fouls, fouls>>
 
 If a robot other than the keeper touches the ball while being partially inside its own defense area, the game is stopped, the robot receives a <<Yellow Card, yellow card>> and the opponent team resumes the game with a <<Direct Free Kick, direct free kick>>. The foul counter is not increased.
 
-If a robot other than the keeper touches the ball while being entirely inside its own defense area, the game is stopped and a <<Penalty Kick, penalty kick>> is awarded to the other team. The foul counter is not increased.
+If a robot other than the keeper touches the ball while being entirely
+inside its own defense area, the game is stopped and a <<Red Card, red
```

Comment:
in #14 it was suggested/decided to keep the penalty kick (which is turned into a one-on-one kick) for the multiple defender foul.